### PR TITLE
DM: add v1.0.2 release notes

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

Add v1.0.2 release notes

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1995/files
### Which version does your change affect? 

3.0/3.1/2.1/dev

### CODE REVIEW

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+DM version: 1.0.2
+
+DM-Ansible version: 1.0.2
+
+## Improvements
+
+- Generate some config items for DM-worker automatically
+- Generate some config items for replication task automatically
+- Simplify the output of query-status without arguments
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
- Simplify the output of `query-status` without arguments
```

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## Bug fixes
+
+- Fix some panic when starting up or executing SQL statements
+- Fix abnormal sharding DDL replication on DDL execution timeout
+- Fix starting task failure caused by the checking timeout or any inaccessible DM-worker
+- Fix SQL execution retry for some error
+
+## Detailed Bug Fixes and Changes
+
+- Generate random server-id for DM-worker config automatically [#337](https://github.com/pingcap/dm/pull/337)
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
- Generate random `server-id` for DM-worker config automatically [#337](https://github.com/pingcap/dm/pull/337)
```

How about quoting (``` ` ```) the words that have been quoted in https://github.com/pingcap/docs-cn/pull/1995/files.

Comment (yikeke) [reply dev/reference/tools/data-migration/releases/1.0.2.md]: Thanks for reminding me. Updated now.

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Manage DB connections directly for downstream
+
+## Bug fixes
+
+- Fix some panic when starting up or executing SQL statements
+- Fix abnormal sharding DDL replication on DDL execution timeout
+- Fix starting task failure caused by the checking timeout or any inaccessible DM-worker
+- Fix SQL execution retry for some error
+
+## Detailed Bug Fixes and Changes
```
Comment (lilin90) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
## Detailed bug fixes and changes
```

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- Generate random `server-id` for DM-worker config automatically [#337](https://github.com/pingcap/dm/pull/337)
+- Generate `flavor` for DM-worker config automatically [#328](https://github.com/pingcap/dm/pull/328)
+- Generate `relay-binlog-name` and `relay-binlog-gtid` for DM-worker config automatically [#318](https://github.com/pingcap/dm/pull/318)
+- Generate table name list for dumping in task config from black & white table lists automatically [#326](https://github.com/pingcap/dm/pull/326)
```
Comment (lilin90) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
- Generate the name list of tables to be dumped in task config from black & white table lists automatically [#326](https://github.com/pingcap/dm/pull/326)
```

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- Fix DM-worker panic when committing a DML transaction failed [#313](https://github.com/pingcap/dm/pull/313)
+- Fix DM-worker or DM-master panic when the listening port is being used [#301](https://github.com/pingcap/dm/pull/301)
+- Fix retry for error code 1105 [#321](https://github.com/pingcap/dm/pull/321), [#332](https://github.com/pingcap/dm/pull/332)
+- Fix retry for `Duplicate entry` and `Data too long for column` [#313](https://github.com/pingcap/dm/pull/313)
+- Fix task check timeout when having large amounts of tables in upstream [#327](https://github.com/pingcap/dm/pull/327)
+- Fix starting task failure when any DM-worker is not accessible [#319](https://github.com/pingcap/dm/pull/319)
+- Fix potential DM-worker startup failure in GTID mode after being recovered from corrupt relay log [#339](https://github.com/pingcap/dm/pull/339)
+- Fix in-memory TPS count for sync unit [#294](https://github.com/pingcap/dm/pull/294)
+- Manage DB connections directly for downstream [#325](https://github.com/pingcap/dm/pull/325)
+- Improve error system by refining error information passed between components [#320](https://github.com/pingcap/dm/pull/320)
```
Comment (lilin90) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
- Improve the error system by refining error information passed between components [#320](https://github.com/pingcap/dm/pull/320)
```

**Pull request merged**